lib/active_record/acts_as/version.rb

```diff
@@ -1,5 +1,5 @@
 module ActiveRecord
   module ActsAs
-    VERSION = "3.0.1"
+    VERSION = "5.2.0.1"
```

Comment:
Where are we getting this version number from?

Reply:
changed into 4.0.0
The reason I put it as such was because the upstream repo's latest version is 5.2.0 and they also have their own 4.0.0, so I am thinking this might be confusing later on (we are relying on their repo for this one)